README.md

```diff
@@ -1,147 +1,141 @@
-**README**
+# Pyronear Dev Environment
 
-This Docker Compose configuration sets up a development environment for Pyronear's API along with supporting services like a PostgreSQL database, MinIO for S3 emulation, Pyro Engine, and Promtail for log shipping.
-
-## Services
-1. **pyro-api**: Runs the Pyronear API using uvicorn.
-2. **db**: PostgreSQL database for the API.
-3. **minio**: Emulates AWS S3 using [MinIO](https://github.com/minio/minio).
-4. **pyro-engine**: Pyro Engine service.
-5. **reolinkdev**: a service which imitate a reolink camera by sending back pictures of fire.
-6. **frontend**: our webapp available on the 8085 port.
+This repository provides a Docker Compose configuration to run a full Pyronear development environment with the API, database, S3 emulation, frontend, notebooks, and optional camera engine.
 
+---
 
-   _Additional services (helpers):_
-7. **notebooks** : Python notebook server to run scripts on api without having to install python
-8. **db-ui** (pgadmin): UI to visualize and manipulate the data in PostgreSQL database
+## ⚙️ Installation
 
-## Installation
 ### Prerequisites
-- Docker and Docker Compose installed on your system.
-- Precommit hook installed on this repo
 
-### Running everything (engine + predefined alerts)
+* Docker and Docker Compose
+* Add this line to `/etc/hosts` so the MinIO endpoint resolves correctly:
```

Comment:
I do not know where to put this command as it is necessary only if you want to read data from front in a browser. Might be okay here